front/app/components/ConsentManager/index.tsx

```diff
@@ -125,87 +155,81 @@ const ConsentManager = () => {
 
     setPreferences(newPreferences);
     saveConsent(newPreferences);
+    setScreen(null);
   };
 
-  const toggleDefault = (modalOpened: boolean) => {
```

Comment:
Only still used when `modalOpened` was false (see below), so removed this argument.